Sources/Core/Sources/Network/Protocol/Packets/Play/Clientbound/EntityAttributesPacket.swift

```diff
@@ -1,8 +1,20 @@
 import Foundation
 
 public enum EntityAttributesPacketError: LocalizedError {
-  case invalidModifierOperation(UInt8)
+  case invalidModifierOperationRawValue(UInt8)
   case invalidAttributeKey(String)
+  
+  public var errorDescription: String? {
+    switch self {
+    case .invalidModifierOperationRawValue(let rawValue):
+      return """
+      Invalid modifier operation.
+      Raw value: \(rawValue)
+      """
+    case .invalidAttributeKey(let attributeKey):
+      return "Invalid attribute key: \(attributeKey)"
```

Comment:
nvm, I see what you've done, wherever you use a colon followed by an inserted value you don't use a full stop (I just flagged it because it was a one-line message without a full stop but your way is more consistent). That's good 👍